sunkit_image/radial.py

```diff
@@ -577,3 +583,111 @@ def fnrgf(
         data[annulus] = ratio_mix[0] * smap.data[annulus] + ratio_mix[1] * data[annulus]
 
     return sunpy.map.Map(data, smap.meta)
+
+
+def rhef(
```

Comment:
This will need one of these https://docs.astropy.org/en/stable/api/astropy.units.quantity_input.html#astropy.units.quantity_input for application radius and vignette.